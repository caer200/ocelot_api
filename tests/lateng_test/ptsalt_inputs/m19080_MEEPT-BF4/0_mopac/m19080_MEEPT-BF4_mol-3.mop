CHARGES

 
B	4.408442 +1 2.187308 +1 3.022269 +1
F	4.719077 +1 2.804957 +1 1.811107 +1
F	5.533511 +1 2.122070 +1 3.822616 +1
F	3.404012 +1 2.926443 +1 3.651541 +1
F	3.939170 +1 0.898417 +1 2.760262 +1
