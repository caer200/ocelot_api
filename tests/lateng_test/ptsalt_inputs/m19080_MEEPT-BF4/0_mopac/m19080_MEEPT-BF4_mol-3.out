 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:07:50 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       B          4.40844200  *   2.18730800  *   3.02226900  *
     2       F          4.71907700  *   2.80495700  *   1.81110700  *
     3       F          5.53351100  *   2.12207000  *   3.82261600  *
     4       F          3.40401200  *   2.92644300  *   3.65154100  *
     5       F          3.93917000  *   0.89841700  *   2.76026200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         B          4.4084      2.1873      3.0223
     2         F          4.7191      2.8050      1.8111
     3         F          5.5335      2.1221      3.8226
     4         F          3.4040      2.9264      3.6515
     5         F          3.9392      0.8984      2.7603

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: B F4  =     5 atoms



      MOLECULAR POINT GROUP   :   C2  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1          B      2      3      4      5
      2          F      1
      3          F      1
      4          F      1
      5          F      1

     Ion Atom No.  Type    Charge
 
    1       1      B       -1

          COMPUTED CHARGE ON SYSTEM: -1

          COMPUTED CHARGE ON SYSTEM:  -1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
